src/sections/Company/Brand/Brand-components/catalog.js

```diff
@@ -30,7 +30,7 @@ const Catalog = () => {
             <h1 className="layerH3">Catalog</h1>
           </Col>
           <Col xs={12} sm={6} className="download-button">
-            <a href="/brand/catalog-brand-kit.zip">
+            <a href="/brand/catalog.zip">
```

Comment:
Thanks! Will you please use the existing filename, so that this package remains consistently named alongside the others?

Reply:
Yes, just to make sure, do we need to replace the existing icons or add these extra ones?

Reply:
Done, by adding the new icons to the existing ones.